Common/Types/test/valuepack.cxx

```diff
@@ -23,9 +23,34 @@
 #include <bitset>
 #include "../include/Types/ValuePack.h"
 
+using TwoFourNinePack = o2::types::ValuePack<unsigned int, 2, 4, 9>;
+
+union Foo {
+  unsigned int value;
+private:
+  struct {
+    unsigned int x:2;
+    unsigned int y:4;
+    unsigned int z:9;
+  };
+public:
+  unsigned int getX() const { return x; }
+  unsigned int getY() const { return y; }
+  unsigned int getZ() const { return z; }
+};
```

Comment:
This is the union approach; From here, some advantages for ValuePack are
a) automatic generation of getters/setters
b) automatic generation of constructor and encapsulation